data/crc.yml

```
@@ -19,6 +19,9 @@ crc:
     "Red Hat logo":
       selectors:
         - '{} img[src="/apps/chrome/js/fonts/logo.svg"]'
+    "Purple feedback tab":
+      selectors:
+        - '.pf-c-button[data-ouia-component-id="feedback-button"]'
```

We should start trying to just use the data-ouia-component-id if it exists

```suggestion
        - '[data-ouia-component-id="feedback-button"]'
```